src/default_daemon_config.cpp

```diff
@@ -380,15 +380,15 @@ repowerd::DefaultDaemonConfig::the_backlight()
 {
     if (!backlight)
     {
-        try
+        /*try
```

Comment:
You can remove code that is not used anymore instead of commenting it out. We have a Git history, its easy to bring back if needed. And if its not needed for some months we will not know anymore what it does. After that time its really pointless ;)